src/module_1/module_1_meteo_api.py

```diff
@@ -70,16 +84,25 @@ def get_data_meteo_api(city, from_year, until_year):
     print("You reached the number of maximum attempts. Stopping the execution")
     sys.exit()
 
+
 def clean_data(raw_data):
+    """
+    (Not used)
+    Function defined for cleaning None values inside a dictionary
+    """
     cleaned_data = {}
     for key, value in raw_data.items():
         if value is not None:
             if isinstance(value, dict):
```

Comment:
I think `type(value) is dict` is better here. This logic is not exactly the same as your `isinstance(value, dict)` since the later would be True also in the case that the class of `value` extends the class `dict`, but this doesn't make much sense in this context